src/lib/y2storage/devicegraph.rb

```diff
@@ -90,6 +90,13 @@ class Devicegraph
     #   @return [Device]
     storage_forward :find_device, as: "Device"
 
+    # @!method remove_device(device)
+    #   Removes the device from the devicegraph
```

Comment:
This documentation should be improved to alert against its usage. Right now, the documentation in `libstorage-ng` is better and more explicit than this forwarded method. That should not happen in general.